apps/openmw/mwmechanics/pathfinding.cpp

```diff
@@ -314,17 +314,14 @@ namespace MWMechanics
         {
             const ESM::Pathgrid::Point oldStart(*getPath().begin());
             buildPath(startPoint, endPoint, cell, pathgridGraph);
-            if (mPath.size() >= 2)
```

Comment:
This check doesn't look redundant to me because buildPath() modifies mPath at which point it could have a totally different size.